tools/sapp/sapp/tests/interactive_test.py

```diff
@@ -251,6 +257,119 @@ def testListIssuesFilterFilenames(self):
+        stderr = self.stderr.getvalue().strip()
+        self.assertIn("'max_trace_length_to_sinks' should be", stderr)
+        self._clear_stdout()
+
+        self.interactive.issues(
+            exact_trace_length_to_sources=1, max_trace_length_to_sources=1
+        )
+        stderr = self.stderr.getvalue().strip()
+        self.assertIn("can't be set together", stderr)
+        self._clear_stdout()
+
+        self.interactive.issues(
+            exact_trace_length_to_sinks=1, max_trace_length_to_sinks=1
+        )
+        stderr = self.stderr.getvalue().strip()
+        self.assertIn("can't be set together", stderr)
+        self._clear_stdout()
+
+        self.interactive.issues(exact_trace_length_to_sources=1)
+        output = self.stdout.getvalue().strip()
+        self.assertIn("Issue 1", output)
+        self.assertNotIn("Issue 2", output)
+        self.assertNotIn("Issue 3", output)
+        self._clear_stdout()
+
+        self.interactive.issues(exact_trace_length_to_sinks=1)
+        output = self.stdout.getvalue().strip()
+        self.assertIn("Issue 1", output)
+        self.assertNotIn("Issue 2", output)
+        self.assertNotIn("Issue 3", output)
+        self._clear_stdout()
+
+        self.interactive.issues(exact_trace_length_to_sources=[1, 2])
+        output = self.stdout.getvalue().strip()
+        self.assertIn("Issue 1", output)
+        self.assertIn("Issue 2", output)
+        self.assertNotIn("Issue 3", output)
+        self._clear_stdout()
+
+        self.interactive.issues(exact_trace_length_to_sinks=[1, 2])
+        output = self.stdout.getvalue().strip()
+        self.assertIn("Issue 1", output)
+        self.assertIn("Issue 2", output)
+        self.assertNotIn("Issue 3", output)
+        self._clear_stdout()
+
+        self.interactive.issues(max_trace_length_to_sources=1)
+        output = self.stdout.getvalue().strip()
+        self.assertIn("Issue 1", output)
+        self.assertNotIn("Issue 2", output)
+        self.assertNotIn("Issue 3", output)
+        self._clear_stdout()
+
+        self.interactive.issues(max_trace_length_to_sinks=1)
+        output = self.stdout.getvalue().strip()
+        self.assertIn("Issue 1", output)
+        self.assertNotIn("Issue 2", output)
+        self.assertNotIn("Issue 3", output)
+        self._clear_stdout()
+
+        self.interactive.issues(max_trace_length_to_sources=2)
+        output = self.stdout.getvalue().strip()
+        self.assertIn("Issue 1", output)
+        self.assertIn("Issue 2", output)
+        self.assertNotIn("Issue 3", output)
+        self._clear_stdout()
+
+        self.interactive.issues(max_trace_length_to_sinks=2)
+        output = self.stdout.getvalue().strip()
+        self.assertIn("Issue 1", output)
+        self.assertIn("Issue 2", output)
+        self.assertNotIn("Issue 3", output)
+        self._clear_stdout()
+
+        self.interactive.issues(
+            max_trace_length_to_sources=1, max_trace_length_to_sinks=1,
```

Comment:
`black` [strikes again](https://our.internmc.facebook.com/intern/diff/D19503869/). Looks like it doesn't like the trailing comas